pdsspect/roi.py

```diff
@@ -68,35 +143,25 @@ def lock_coords_to_pixel(self, data_x, data_y):
             The corresponding y pixel coordinate
         """
 
-        point_x = point_y = None
-
-        # Set default values for data points outside the pan
-        if data_x <= 0:
-            point_x = -.5
-        if data_y <= 0:
-            point_y = -.5
-        if data_x >= (self.image_set.x_radius * 2 - 1):
-            point_x = (self.image_set.x_radius * 2 - 1.5)
-        if data_y >= (self.image_set.y_radius * 2 - 1):
-            point_y = self.image_set.y_radius * 2 - 1.5
+        point_x, point_y = self._get_default_data_values(data_x, data_y)
 
         if None not in (point_x, point_y):
             return point_x, point_y
 
-        X, Y = np.ceil(data_x), np.ceil(data_y)
-        x, y = np.floor(data_x), np.floor(data_y)
+        X, Y = np.ceil((data_x, data_y))
+        x, y = np.floor((data_x, data_y))
```

Comment:
I realize this usage of X, Y, x, and y is not new in this commit, but having variables differ by capitalization does not really convey any meaning and can result in confusion ... the difference in capitalization means something to you but I can't tell what it means.  Maybe something like `x_high`, `x_low`?